### DISCUSSION TITLE
What is the most natural sounding TTS?

### DISCUSSION THREAD

**Discussion by Darin755** [organicmaps/organicmaps#8423] [Q&A]
I am looking for a TTS that is on F-droid and locally run that sounds a bit more natural. I am right now using espeak but it is fairly robotic.

Any suggestions?

**Comment by Urvish79** [comment]
You can use eSpeak-ng(new generation) unlike its older version eSpeak which sounds more robotic this is fairly natural. It's a compact, open-source speech synthesizer that supports multiple languages and can run on various platforms. It might be worth trying out to see if it meets your requirements.
